docs/java/release-notes.md

```diff
@@ -11,7 +11,7 @@ keywords:
 - cloud sdk
 - sap cloud sdk
 ---
-
+<!--
```

Comment:
Currently the release notes for 3.23.0 are online which might confuse consumers because the version is not yet on Maven Central. So I just take them offline for now.